src/components/SpecialResolution/CreateSpecialResolution.vue

```diff
@@ -384,40 +390,45 @@ export default class CreateSpecialResolution extends Mixins(DateMixin) {
   }
 
   /** called to store signing party. */
-   @Watch('signatory', { deep: true })
-  protected async onsignatoryChanged (): Promise<void> {
+  @Watch('signatory', { deep: true })
+  protected async onSignatoryChanged (): Promise<void> {
     this.setResolution({
-      ...this.getcreateResolution,
+      ...this.getCreateResolution,
       signatory: this.signatory
     })
     // wait for store update
     await this.$nextTick()
     await this.validate()
   }
 
-   /** called to store component validity to store. */
-   protected async validate () {
-     // wait to reflect validation state
-     await this.$nextTick()
+  /** called to store component validity to store. */
+  protected async validate () {
+    // wait to reflect validation state
+    await this.$nextTick()
 
-     // date validation
-     const isResolutionDateValid = this.$refs?.resolutionDatePickerRef?.isDateValid()
-     const isSignatureDateValid = this.$refs?.signatureDatePickerRef?.isDateValid()
+    // date validation
+    const isResolutionDateValid = this.$refs?.resolutionDatePickerRef?.isDateValid()
+    const isSignatureDateValid = this.$refs?.signatureDatePickerRef?.isDateValid()
 
-     const isFormValid = this.formValid && isResolutionDateValid && isSignatureDateValid
-     // setting component validity flag
-     this.setValidComponent({ key: 'isValidCreateSpecialResolution', value: isFormValid })
-   }
+    const isFormValid = this.formValid && isResolutionDateValid && isSignatureDateValid
+    // setting component validity flag
+    this.setValidComponent({ key: 'isValidCreateSpecialResolution', value: isFormValid })
+  }
 
-   /** Set values if exist
-    * while coming back from summary page this form need to show existing values.
-   */
-   protected created () {
-     this.signatory = this.getcreateResolution.signatory || { ...EmptySigningPerson }
-     this.resolution = this.getcreateResolution.resolution || ''
-     this.resolutionDateText = this.getcreateResolution.resolutionDate || ''
-     this.signingDate = this.getcreateResolution.signingDate || ''
-   }
+  /** Set values if exist
+   * while coming back from summary page this form need to show existing values.
+  */
+  protected created () {
+    this.signatory = this.getCreateResolution.signatory ||
+      {
+        givenName: '',
+        familyName: '',
+        additionalName: null
+      }
+    this.resolution = this.getCreateResolution.resolution || ''
+    this.resolutionDateText = this.getCreateResolution.resolutionDate || ''
+    this.signingDate = this.getCreateResolution.signingDate || ''
+  }
```

Comment:
FYI, getting the data here (from getters) works because this component isn't created until after the data is loaded. I think this assumption is solid, but if there's any reason the getter might have a different value after the component is created/mounted then you'll need to watch the getter.

No action needed here, just FYI.